doc/en/multitest.rst

```diff
@@ -1011,8 +1011,10 @@ However you can provide custom name generation functions to override this
 functionality via ``@testcase(parameters=..., name_func=custom_name_func)``
 syntax. You just need to implement a function that accepts
 ``func_name`` and ``kwargs`` as arguments, ``func_name`` being a string and
-``kwargs`` being an ``OrderedDict``.
-See
+``kwargs`` being an ``OrderedDict``. You may check if there is an argument
+``for_display`` in `kwargs``, if the value is true, then it is suggested to
```

Comment:
I agree with Ryan, I don't think we need another for_display here. If this is the display name we discussed, it does not belong here (parameterization session).